fog/view/enclave/api/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub enum ViewEnclaveRequest {
     /// An encrypted fog_types::view::QueryRequest
     /// Respond with fog_types::view::QueryResponse
     Query(EnclaveMessage<ClientSession>, UntrustedQueryResponse),
+    /// An encrypted fog_types::view::QueryRequest
+    /// Respond with fog_types::view::QueryResponse.
+    QueryBackend(EnclaveMessage<NonceSession>, UntrustedQueryResponse),
```

Comment:
I think this may be the source of Nick's comment later, this should probably be called `QueryStore` to match the `query_store` method name...?

Reply:
Done.